apps/www/content/docs/installation/vite.mdx

```diff
@@ -48,7 +48,6 @@ Add the `baseUrl` and `paths` properties to the `compilerOptions` section of the
     }
   ],
   "compilerOptions": {
-    "baseUrl": ".",
     "paths": {
       "@/*": ["./src/*"]
```

Comment:
By the way, why does the "solution"/"workspace root" `tsconfig.json` need to have this configured? What depends on the options in this `tsconfig.json`? I would think only `tsconfig.app.json` needs to be updated.